assets/src/users/UserContext.tsx

```diff
@@ -48,16 +55,21 @@ const useAuthUser = () => {
   const { user } = useAuth0();
   return user;
 };
-const usePayUser = () => React.useContext(PayUserContext);
+const usePayUser = () => React.useContext(PayUserContext)!;
 const useUser = () => React.useContext(UserContext);
 
 const PayUserProvider: React.FC<{}> = ({ children }) => {
-  const query = useCheckAuthQuery({});
-  const value =
-    query.data && query.data.me ? fromGQLUser(query.data.me) : undefined;
-
+  let isLoading = false;
+  const { isInitializing } = useAuth0();
+  const { loading, data } = useCheckAuthQuery({});
+  if (isInitializing || loading) {
+    isLoading = true;
+  }
+  const user = data && data.me ? fromGQLUser(data.me) : undefined;
   return (
-    <PayUserContext.Provider value={value}>{children}</PayUserContext.Provider>
+    <PayUserContext.Provider value={{ user, isLoading }}>
```

Comment:
@acodegirl perhaps it is better to set the value to `{ user, isLoading: isInitializing || loading }` directly. Then you can remove the mutable `isLoading` local variable state.